depthai_nodes/ml/parsers/mediapipe_palm_detection.py

```diff
@@ -48,15 +51,42 @@ def __init__(self, conf_threshold=0.5, iou_threshold=0.5, max_det=100, scale=192
         @param max_det: Maximum number of detections to keep.
         @type max_det: int
         """
-        dai.node.ThreadedHostNode.__init__(self)
-        self.input = self.createInput()
-        self.out = self.createOutput()
+        super().__init__()
 
         self.conf_threshold = conf_threshold
         self.iou_threshold = iou_threshold
         self.max_det = max_det
         self.scale = scale
 
+    def build(
+        self,
+        head_config: Dict[str, Any],
+    ):
+        """Sets the head configuration for the parser.
+
+        Attributes
+        ----------
+        head_config : Dict
+            The head configuration for the parser.
+
+        Returns
+        -------
+        MPPalmDetectionParser
+            Returns the parser object with the head configuration set.
+        """
+
+        output_layers = head_config["outputs"]
```

Comment:
Should save to `self.output_layers` attribute and add this attribute in the initializer